src/_pytest/python.py

```diff
@@ -942,6 +960,24 @@ def make_unique_test_ids(self) -> List[str]:
 
         return resolved_ids
 
+    def make_parameter_keys(self) -> Iterable[Dict[str, Hashable]]:
+        """Make hashable keys for parameters in a parametrized test.
+
+        This key will be considered to determine if parameters
```

Comment:
This key will be considered (along with the parameter name) to determine ...

Reply:
> (along with the parameter name)

What is the "parameter name"? `reorder_items` uses e.g. `key = (argname, param_key, item.path, item_cls)`, and `FixtureDef.cache_key` uses `param_key`. Do you mean argument name?